extensions/json-schema/services/src/main/resources/META-INF/cxs/schemas/items/interests/interests.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$id": "https://unomi.apache.org/schemas/json/interests/1-0-0",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "self":{
+    "vendor":"org.apache.unomi",
+    "name": "interests",
+    "format":"jsonschema",
+    "version":"1-0-0"
+  },
+  "title": "Interests",
+  "type": "object",
+  "patternProperties": {
+    "^.*$": {
```

Comment:
Can't we limit to only alphanumerics characters?

Reply:
This might cause problems in other languages such as Chinese, Cyrillic, etc...

Reply:
For now we will not restreint this, if validation is asked on the keys of the interests we will do something, for now it's not the case.